control/controlclient/map.go

```diff
@@ -259,6 +260,16 @@ func undeltaPeers(mapRes *tailcfg.MapResponse, prev []*tailcfg.Node) {
 				n.Online = &online
 			}
 		}
+		if ec := mapRes.EndpointChange; ec != nil {
+			if n, ok := peerByID[ec.ID]; ok {
+				if ec.DERPRegion != 0 {
```

Comment:
no way to say empty?

Reply:
kinda pointless. we have Offline already